plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -1200,6 +1202,39 @@ public void testAllTypesInsert()
         }
     }
 
+    @Test
+    public void testInetInsert()
+    {
+        try (TestCassandraTable testCassandraTable = testTable(
+                "table_inet_insert",
+                ImmutableList.of(
+                        partitionColumn("key", "text"),
+                        generalColumn("typeinet", "inet")),
+                ImmutableList.of())) {
+            String sql = "SELECT key, typeinet" +
+                    " FROM " + testCassandraTable.getTableName();
+            assertEquals(execute(sql).getRowCount(), 0);
+
+            // TODO Following types are not supported now. We need to change null into the value after fixing it
+            // blob, frozen<set<type>>, inet, list<type>, map<type,type>, set<type>, decimal, varint
+            // timestamp can be inserted but the expected and actual values are not same
+            execute("INSERT INTO " + testCassandraTable.getTableName() + " (" +
+                    "key," +
+                    "typeinet" +
+                    ") VALUES (" +
+                    "'key1', " +
+                    "IPADDRESS '10.10.10.1'" +
```

Comment:
Please add more test cases, e.g. min, max, null, ipv6.

Reply:
@ebyhr Just found out there is already a test dedicated to testing all the edge cases of the `INET` type: `TestCassandraTypeMapping.testCassandraInet`.

I removed this `TestCassandraConnectorTest.testInetInsert` altogether because it's a temporary test I added to get fast feedback. It was extracted from `TestCassandraConnectorTest.testAllTypesInsert` so it's already covered and was pushed by accident.